spec/controllers/v1/item_types_controller_spec.rb

```diff
@@ -22,6 +22,11 @@
             'allowed_keys' => allowed_keys,
             'items' => [])
         end
+        it 'creates Permission for creator' do
+          submit
+          json = JSON.parse response.body
+          expect(Permission.where(item_type_id: json['id'], service: @service).count).not_to eq 0
```

Comment:
Better to do

```ruby
new_item_type = ItemType.find(json.fetch 'id')
expect(service).to be_able_to_write_to new_item_type
```

([example](https://github.com/umts/aggressive-epsilon/blob/1e69a36cf84d908c73b7b5283c5c25351103501a/spec/models/service_spec.rb#L53))